linien/server/registers.py

```diff
@@ -89,9 +89,10 @@ def write_registers(self):
             # of stopping the sweep run after `request_lock` is set by setting
             # `sweep.clear`
             logic_sweep_run=1,
+            logic_sweep_pause=int(params["sweep_pause"]),
+            logic_sweep_pause_value=int(params["sweep_center"] * 8191),
             logic_sweep_step=int(
-                int(params["sweep"])  # controls whether sweep is turned on
-                * DEFAULT_SWEEP_SPEED
+                DEFAULT_SWEEP_SPEED
```

Comment:
why did you delete int(params["sweep"])?

Reply:
I completely removed the "sweep" parameter and introduced  "pause_sweep" instead. I think it is clearer what the parameter does now. I removed ´params["sweep"]´ here since it was the old, buggy way to pause the sweep: It just stopped at whatever position it was.

Reply:
ah, makes totally sense :+1: